packages/contracts/src/test/PlayerHandle.t.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+import "../PlayerHandle.sol";
+
+contract MockENSResolver is IENSResolver {
+    mapping(address => string) private ensNames;
+
+    function setEnsName(address addr, string memory name) external {
+        ensNames[addr] = name;
+    }
+
+    function getEnsName(address addr) external view override returns (string memory) {
+        return ensNames[addr];
+    }
+}
```

Comment:
Could you spin this out in a separate file (in the test repo), and add a deployment of this + `PlayerHandle` to the `Deploy.s.sol`? You'll also want to add an entry for `PlayerHandle` in `deployment.ts` on the frontend.